modules/beetmover_scriptworker/templates/script_config.json.erb

```diff
@@ -36,13 +63,13 @@
             "buckets": <%= require "json"; JSON.pretty_generate(@env_config["dep_partner_buckets"]) %>
         },
 <% end -%>
-<% if @env_config["prod_maven_buckets"] -%>
-        "maven-production": {
+<% if @env_config["dep_maven_buckets"] -%>
+        "maven-staging": {
             "credentials": {
-                "id": "<%= @env_config["prod_maven_beetmover_aws_access_key_id"] %>",
-                "key": "<%= @env_config["prod_maven_beetmover_aws_secret_access_key"] %>"
+                "id": "<%= @env_config["dep_maven_beetmover_aws_access_key_id"] %>",
+                "key": "<%= @env_config["dep_maven_beetmover_aws_secret_access_key"] %>"
             },
-            "buckets": <%= require "json"; JSON.pretty_generate(@env_config["prod_maven_buckets"]) %>
+            "buckets": <%= require "json"; JSON.pretty_generate(@env_config["dep_maven_buckets"]) %>
         },
 <% end -%>
         "dep": {
```

Comment:
To me, it would be cleaner to store the whole data in a dictionary (aka a "hash" in puppet) and let Puppet transform it into JSON. For instance: https://github.com/mozilla-releng/build-puppet/blob/1053dec537460ba36399787ba2edca30d06448db/modules/pushapk_scriptworker/manifests/init.pp#L76. The sad counterpart of this call is that Puppet is too dumb to understand when the data doesn't change from a run to another. I know some contributors put an addon online to [solve this problem](https://www.halkeye.net/2012/03/14/output-pretty-json-puppet/), but I haven't looked into it further.